config/config.go

```diff
@@ -56,6 +56,9 @@ var (
 		"analyzer.topology.backend":            {"graph.backend"},
 		"analyzer.topology.k8s.config_file":    {"k8s.config_file"},
 		"analyzer.topology.k8s.probes":         {"k8s.probes"},
+		"agent.topology.sflow.bind_address":    {"sflow.bind_address"},
```

Comment:
topology ?

Reply:
This one is doing both but better to move it in the flow section as rely on capture